daemon/cdi.go

```diff
@@ -0,0 +1,42 @@
+package daemon
+
+import (
+	"fmt"
+
+	"github.com/container-orchestrated-devices/container-device-interface/pkg/cdi"
+	specs "github.com/opencontainers/runtime-spec/specs-go"
+	"github.com/sirupsen/logrus"
+)
+
+// initializeCdi initializes the CDI registry and registers the CDI device driver.
+func initializeCdi(d *Daemon) error {
```

Comment:
Spell it `CDI` or `cdi`, never `Cdi`. [This isn't Java code.](https://github.com/golang/go/wiki/CodeReviewComments#initialisms)

Reply:
OK. Will to.

Reply:
One question: Does this convention hold for types too? I see for example `DaemonCli` as a type instead of `DaemonCDI`, for example.

Reply:
Thanks. I'll keep that in mind.